src/components/Link/ArrowLink/ArrowLink.tsx

```diff
@@ -0,0 +1,71 @@
+//- React Imports
```

Comment:
Not your fault obviously, but this component is actually pretty weird. We should simplify this down a lot. If it's going to be a link it should probably have a similar/identical interface to HTML `a` elements.

Reply:
same, never changed this component